js/modules/loki_app_dot_net_api.js

```diff
@@ -98,7 +229,11 @@ const sendToProxy = async (
   payloadObj.body = false; // free memory
 
   // make temporary key for this request/response
-  const ephemeralKey = await libsignal.Curve.async.generateKeyPair();
+  // const ephemeralKey = await libsignal.Curve.generateKeyPair(); // sync
+  // async maybe preferable to avoid cpu spikes
+  // tho I think sync might be more apt in certain cases here...
+  // like sending
+  const ephemeralKey = await libloki.crypto.generateEphemeralKeyPair();
```

Comment:
`libloki.crypto.generateEphemeralKeyPair` internally uses `libsignal.Curve.async.generateKeyPair()` and slices of `0x05` prefix. it this intended?

Reply:
I've tested it and makes no difference

Reply:
Ok 👍